<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileBrowser</class>
 <widget class="QWidget" name="FileBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fi&amp;les</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="comboBoxDirectories"/>
       </item>
       <item>
        <widget class="QToolButton" name="buttonAddDirectory">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonRemoveDirectory">
         <property name="text">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelDirectory">
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="FileBrowser::DirectoryTreeView" name="treeViewDirectories">
         <property name="contextMenuPolicy">
          <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
         </property>
         <property name="indentation">
          <number>10</number>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,0,0,0,0,0,0,2">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="buttonRefresh">
           <property name="text">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="buttonMarkComplete">
           <property name="text">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="buttonExclude"/>
         </item>
         <item>
          <widget class="QToolButton" name="buttonShowExcluded"/>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionShowExcluded">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../cmake-build-relwithdebinfo/src/icons.qrc">
     <normaloff>:/icons/bx-hide.svg</normaloff>
     <normalon>:/icons/bx-show.svg</normalon>:/icons/bx-hide.svg</iconset>
   </property>
   <property name="text">
    <string>Show excluded files</string>
   </property>
   <property name="toolTip">
    <string>Show excluded files</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExclude">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../cmake-build-relwithdebinfo/src/icons.qrc">
     <normaloff>:/icons/bx-block.svg</normaloff>:/icons/bx-block.svg</iconset>
   </property>
   <property name="text">
    <string>Exclude file from project</string>
   </property>
   <property name="toolTip">
    <string>Exclude file from project</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../cmake-build-relwithdebinfo/src/icons.qrc">
     <normaloff>:/icons/bx-refresh.svg</normaloff>:/icons/bx-refresh.svg</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh complete directory tree</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionMarkComplete">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../cmake-build-relwithdebinfo/src/icons.qrc">
     <normaloff>:/icons/bx-check-circle.svg</normaloff>:/icons/bx-check-circle.svg</iconset>
   </property>
   <property name="text">
    <string>Mark as complete</string>
   </property>
   <property name="toolTip">
    <string>Mark as complete</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddDirectory">
   <property name="icon">
    <iconset resource="../../cmake-build-relwithdebinfo/src/icons.qrc">
     <normaloff>:/icons/bx-plus-circle.svg</normaloff>:/icons/bx-plus-circle.svg</iconset>
   </property>
   <property name="text">
    <string>Add directory to the project</string>
   </property>
   <property name="toolTip">
    <string>Add directory to the project</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRemoveDirectory">
   <property name="icon">
    <iconset resource="../../cmake-build-relwithdebinfo/src/icons.qrc">
     <normaloff>:/icons/bx-minus-circle.svg</normaloff>:/icons/bx-minus-circle.svg</iconset>
   </property>
   <property name="text">
    <string>Remove directory from the project</string>
   </property>
   <property name="toolTip">
    <string>Remove directory from the project</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileBrowser::DirectoryTreeView</class>
   <extends>QTreeView</extends>
   <header>FileBrowser/DirectoryTreeView.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../cmake-build-relwithdebinfo/src/icons.qrc"/>
 </resources>
 <connections/>
</ui>
